Topic8/src/main/java/com/aguirre/topic8/controllers/CourseController.java

```diff
@@ -0,0 +1,58 @@
+package com.aguirre.topic8.controllers;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import com.aguirre.topic8.models.Course;
+import com.aguirre.topic8.models.Student;
+import com.aguirre.topic8.repositories.CourseDao;
+import com.aguirre.topic8.repositories.StudentCourseDao;
+import com.aguirre.topic8.repositories.StudentDao;
+import com.google.gson.Gson;
+
+@ContextConfiguration
+@RestController
+@RequestMapping("/courses")
+public class CourseController {
+
+	@Autowired
+	private CourseDao courseDao;
+
+	@Autowired
+	private StudentCourseDao studentCourseDao;
+
+	@Autowired
+	private StudentDao studentDao;
+
+	@RequestMapping(value = "/add", method = RequestMethod.POST)
+	public String addCourse(String jsonString) {
+		Gson gson = new Gson();

```

Comment:
Please validate if it works as we expected without the need to use the Gson library. According Spring documentation it works by default using Jsons